### DISCUSSION TITLE
Problems with FSLeyes

### DISCUSSION THREAD

**Discussion by mmsarhan** [muscbridge/PyDKE#274] [Technical Support]
Hello,
I recently was able to download everything needed for pydesigner and get everything set up. The only issue is that FSLeyes does not work. Its keeps saying "command not found" I was able to download fslview and that works with the command fslview_decapulate. Is there a way to download FSLeyes on its own to fix FSL and get it to work?
I am running Centos 7 on a Virtual Machine. Thank you.

**Comment by TheJaeger** [comment]
You may follow the instructions for **Install from conda-forge (recommended)** on [FSLeyes Wiki](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes). This will install standalone FSLeyes within your Python env.

**Reply by mmsarhan** [reply]
Do I run this in my condo enrironment? Or do I run it within the system like how you have to download FSL?

**Reply by TheJaeger** [reply]
You can install this into your base Conda environment

**Reply by mmsarhan** [reply]
@TheJaeger 
I just ran the command and it did not work. Am I doing something wrong for the download not to work? Here is the error that it gave me.

(base) [blab@localhost ~]$ conda install -c conda-forge fsleyes
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): | WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
| WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.*, but conda is ignoring the .* and treating it as 1
\ WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.6.*, but conda is ignoring the .* and treating it as 3.6
done
Solving environment: \ (base) [blab@localhost ~]$ fsleyes
bash: fsleyes: command not found...
(base) [blab@localhost ~]$
